main.js

```diff
@@ -0,0 +1,46 @@
+const todoInput = document.getElementById("searchValue");  // input 입력값
+const todoBtn = document.getElementById("add_btn");  // 추가 버튼
+
+function todoBtnClick() {
+
+    if(!todoInput.value) {
+        alert("Please enter your To Do List ! :-)");  // input에 내용을 입력하지 않고 추가 버튼 누르면 alert 발생
+    
+    } else {
```

Comment:
if-else는 실무에서 사용하는 경우가 매우 드뭅니다.
대신 if-return을 더 많이 사용 하죠.
현재의 경우 

```suggestion
if(!todoInput.value) {
        alert("Please enter your To Do List ! :-)");  // input에 내용을 입력하지 않고 추가 버튼 누르면 alert 발생
return 
}
// 나머지 로직
````
으로 사용하는 것이 더 가독성과 확장성 면에서 좋아보입니다.